src/scancode/cli.py

```diff
@@ -362,6 +363,14 @@ def save_results(scanned_files, format, input, output_file):
     """
     Save results to file or screen.
     """
+    dirname = os.path.dirname(output_file.name)
+    if not os.path.isdir(dirname):
+        try:
+            os.makedirs(dirname)
```

Comment:
Do you think you could use a call to `commoncode.fileutils.create_dir` instead?
See https://github.com/nexB/scancode-toolkit/blob/086addda656150b7feed6da438fe1413091145a2/src/commoncode/fileutils.py#L60

The rationale is that its needs to eventually support long Windows path using the UNC `\\?\` syntax so it is best to control places where directories are created.